internal/bminventory/inventory_v2_handlers.go

```diff
@@ -605,13 +606,63 @@ func (b *bareMetalInventory) signURL(ctx context.Context, infraEnvID, urlString,
 	return urlString, &expiresAt, nil
 }
 
-const ipxeScriptFormat = `#!ipxe
+const ipxeFirstScriptFormat = `#!ipxe
+chain %s&mac=${net0/mac}
+`
+
+const ipxeSecondScriptFormat = `#!ipxe
 initrd --name initrd %s
 kernel %s initrd=initrd coreos.live.rootfs_url=%s random.trust_cpu=on rd.luks.options=discard ignition.firstboot ignition.platform.id=metal console=tty1 console=ttyS1,115200n8 coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"
 boot
 `
 
-func (b *bareMetalInventory) infraEnvIPXEScript(ctx context.Context, infraEnv *common.InfraEnv) (string, error) {
+func (b *bareMetalInventory) firstIPXEScript(infraEnv *common.InfraEnv) (string, error) {
```

Comment:
I think this should have a better name. Maybe `hostRedirectIPXEScript` that way we're describing what the script does in contrast to the one that boots the host rather than just referencing it by order.